src/tests/TotalCard.test.js

```diff
@@ -0,0 +1,12 @@
+import renderer from 'react-test-renderer';
+import TotalCard from '../components/TotalCard';
+
+describe('Filter component', () => {
+  test('Snapshot test', () => {
+    const totalCard = renderer.create(
+      <TotalCard name="AFRICA" total="200987" />,
+    )
```

Comment:
* Kindly perform a mocking data fetching and Redux Store connection as stated in the project [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=You%20may%20need,the%20Redux%20Store.).